### DISCUSSION TITLE
Automatically create ssl certificate with load balancing example

### DISCUSSION THREAD

**Discussion by willhama** [zitadel/zitadel#5086] [Q&A] [answered]
Grüezi Team,

Is the load-balancing example supposed to create a secure connection to my given domain?
https://zitadel.com/docs/self-hosting/deploy/loadbalancing-example

I am running the example on an EC2 instance. Created a reroute from "my.domain" to the elastic IP of the container.

I am able to access but the TRAEFFIK DEFAULT CERT is not valid.

Is this behavior expected?
Is there an easy solution to automatically retrieve a valid cert?

Thanks in advance! <3

**Comment by fforootd** [comment, marked as answer]
Hi There

That example gives you a self-signed certificate.

I think you could use Lets Encrypt with traefik https://doc.traefik.io/traefik/https/acme/#lets-encrypt

**Reply by willhama** [reply]
Okay, I will try it out.

Thanks in advance.

**Reply by fforootd** [reply]
👍
